sourcegraph/sourcegraph
# notebooks: add import and export Markdown functionality

Fixes #28586

## Demo

https://user-images.githubusercontent.com/6417322/154215040-b94528bd-4686-4de1-91dc-b0f7b6dd2565.mov

## Test plan



* Go to the notebooks list page
* Click on the "Import Markdown notebook"
  * You should only be able to select Markdown files
  * You should get an error for files larger than 1MB
* After a successful import, you should be redirected to the newly imported notebook page
* To export a notebook to Markdown, click on the "Export as Markdown" button
  * Your browser should download a Markdown file
